docs/sphinx/reference-core.rst

```diff
@@ -563,6 +563,29 @@ Primary signal/procedure handlers
 
    :return: The primary obs procedure handler
 
+---------------------
+
+.. function:: obs_show_notification(enum obs_notify_type type, const char *text, void *data)
+
+   Show a notification in the main UI.
+
+   :param type: | Can be one of the following types:
+                | OBS_NOTIFY_TYPE_INFO
+                | OBS_NOTIFY_TYPE_WARNING
+                | OBS_NOTIFY_TYPE_ERROR
+   :param text: | Message of notification (append hyperlink at end of message to add action to message)
```

Comment:
```suggestion
   :param text: | Message of notification (append HTML hyperlink at end of message to add action to message)
```

By not specifying that it has to be a HTML hyperlink, developers might get confused when just putting the text itself does nothing.